### PR TITLE
Changed tags to be represented by their own db class, posts no longer sh...

### PR DESCRIPTION
**Pull request by EvanNGoldstein**
...ow 'Tags:' if the post has no tags

### CODE REVIEW

File: app/models/Tag.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from app import db
```
Comment (danrschlosser) [inline app/models/Tag.py]: Let's add a module docstring here

File: app/models/Tag.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from datetime import datetime
+
+now = datetime.now
+
+class Tag(db.Document):
```
Comment (danrschlosser) [inline app/models/Tag.py]: docstring me!

File: app/models/Tag.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+	date_created = db.DateTimeField(default=now, required=True)
+	data_modified = db.DateTimeField(default=now, required=True)
+	tagname = db.StringField(unique=True, max_length=255,required=True)
+
+	def clean(self):
+		self.date_modified = now()
+
+	@classmethod
+	def get_or_create_tags(klass, tagnames):
```
Comment (danrschlosser) [inline app/models/Tag.py]: docstring this and other methods

File: app/models/Tag.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+				tag.save()
+				tagsList.append(tag)
+		return tagsList
+
+
+	def __unicode__(self):
+		return self.tagname
+
+	def __repr__(self):
+		return self.tagname
```
Comment (danrschlosser) [inline app/models/Tag.py]: In this whole file: use spaces, not tabs

File: app/routes/admin/posts.py [hunk 5]
```diff
@@ -94,13 +95,16 @@ def edit(post_id):
...
+
         if form.validate_on_submit():
             post.title = form.title.data
             post.author = User.objects.get(id=ObjectId(form.author.data))
             post.slug = form.slug.data
             post.markdown_content = form.body.data
             post.images = [Image.objects().get(filename=fn) for fn in form.images.data]
-            post.tags = form.tags.data
+            
+
```
Comment (danrschlosser) [inline app/routes/admin/posts.py]: remove extra blank line

File: app/routes/admin/posts.py [hunk 6]
```diff
@@ -179,4 +183,4 @@ def fetch_epiceditor_themes(folder, path):
     :param str folder: The folder that EpicEditor wants.
     :param str path: The path of the file that EpicEditor wants.
     """
-    return send_from_directory(app.static_folder, "css/lib/epiceditor/{folder}/{path}".format(folder=folder, path=path))
+    return send_from_directory(app.static_folder, "css/lib/epiceditor/{folder}/{path}".format(folder=folder, path=path))
```
Comment (danrschlosser) [inline app/routes/admin/posts.py]: We should always have a newline at the end of file

**Comment by EvanNGoldstein** [comment]
Review: @danrschlosser

**Comment by danrschlosser** [comment]
Great!

**Pull request merged**